src/udev/udev-ctrl.c

```diff
@@ -262,7 +262,7 @@ static int ctrl_send(struct udev_ctrl *uctrl, enum udev_ctrl_msg_type type, int
         int err = 0;
 
         memzero(&ctrl_msg_wire, sizeof(struct udev_ctrl_msg_wire));
-        strcpy(ctrl_msg_wire.version, "udev-" VERSION);
+        sprintf(ctrl_msg_wire.version, "%.*s", (int)(sizeof ctrl_msg_wire.version - 1), "udev-" VERSION);
```

Comment:
we might as well use snprintf here, for posterity

Reply:
OK, thanks!

I have also fixed code style of sizeof...